src/components/GameRoom.js

```diff
@@ -78,50 +75,54 @@ function GameRoom({ room, setInRoom, userName, host, gamesWon,_id }) {
       setYouGuessed(true);
     });
 
-    socket.on("all_players_guessed",()=>{setAllPlayersReady(true)})
+    socket.on("all_players_guessed", () => {
+      setAllPlayersReady(true);
+    });
 
-    socket.on("game_over",()=>setGameOver(true))
+    socket.on("game_over", () => setGameOver(true));
 
     //This is at the end of the game
 
     //Returns the winner if it's not yourself
-    socket.on("winner",data=>setWinner(data))
+    socket.on("winner", (data) => setWinner(data));
     //Returned if you are the winner
-    async function patch(){
-      console.log(gamesWon,'games won before PATCH');
+    async function patch() {
+      console.log(gamesWon, "games won before PATCH");
       try {
-        const response = await fetch(`http://localhost:4000/api/v1/user/${_id}`, {
-          method: "PATCH",
-          headers: {
-            "Content-Type": "application/json",
-            Authorization: `Bearer ${token}`,
-          },
-          body: JSON.stringify({
-            gamesWon: ++gamesWon
-          })
-        });
+        const response = await fetch(
+          `http://localhost:4000/api/v1/user/${_id}`,
```

Comment:
Can't use hard-coded server URLs anymore. Remember, we are deployed now. Need to use the base_url variable.